style.css

```diff
@@ -0,0 +1,376 @@
+  justify-content: space-between;
+  padding-top: 10px;
+}
+
+.comment > * {
+  width: 250px;
+  height: auto;
+  background-color: cadetblue;
+  color: white;
+  border: transparent;
+  border-radius: 3px;
+  text-align: center;
+}
+
+.social-icons > * {
+  margin-left: 5px;
+}
+
+nav {
+  background-color: white;
+  border-bottom: solid gray;
+  border-width: 1px;
+  padding: 10px;
+  width: 100%;
+  height: auto;
+  grid-area: nav;
+  display: flex;
+  justify-content: space-evenly;
+  position: fixed;
+}
+
+.nav-button {
+  padding: 3px;
+  max-width: 130px;
+  font-size: 70%;
+  background-color: cadetblue;
+  border-radius: 10px;
+  color: cornsilk;
+  font-family: "Times New Roman", Times, serif;
+}
+
+.onclick {
+  background-color: white;
+  border: transparent;
+  margin-left: 5px;
+}
+
+.nav-search {
+  display: none;
+}
+
+.onclick:focus + .nav-search {
+  display: inline;
+}
+
+#nytimes-logo {
+  font-family: "chomskyregular", Arial, sans-serif;
+  font-size: 100%;
+  font-weight: 400;
+}
+
+section {
+  grid-area: section;
+}
+
+footer {
+  grid-area: footer;
+}
+
+.right-aside {
+  grid-area: r-aside;
+  font-size: 100%;
+  max-width: 100%;
+  float: left;
+}
+
+.title-header {
+  font-style: italic;
+  max-width: 50%;
+  margin: 0 auto;
+}
+
+.main-img,
+.middle-img {
+  max-width: 100%;
+  max-height: 30%;
+  margin-top: 10px;
+}
+
+article {
+  max-width: 600px;
+  margin: 20px auto;
+}
+
+article p {
+  margin: 20px 0;
+}
+
+.diagrams {
+  max-width: 100%;
+  margin: 0;
+}
+
+.diagrams-container {
+  display: grid;
+  grid-template-columns: repeat(4, 4fr);
+  grid-template-rows: repeat(4, 1fr);
+  grid-auto-flow: column;
+  gap: 2px;
+  width: 100%;
+  margin-top: 10px;
+}
+
+.bottom-section img {
+  width: 100%;
+}
+
+.left-b-section img {
+  object-fit: contain;
+}
+
+.r-flex-box > img {
+  max-width: 130px;
+  margin-right: 10px;
+}
+
+.diagrams-container p img {
+  width: 90%;
+}
+
+.bottom-section {
+  width: 100%;
+  margin: auto;
+  border-bottom: 1px solid black;
+  font-size: 100%;
+}
+
+.left-b-section {
+  width: 70%;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  margin-bottom: 20px;
+  font-size: 80%;
+  border-top: solid 1px black;
+}
+
+.left-b-section-1 {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  width: 70%;
+  padding-top: 30px;
+  font-size: 70%;
+  border-top: solid 1px black;
+  margin-top: 20px;
+}
+
+.left-b-section-1 > div {
+  width: 30%;
+}
+
+.left-b-section > div {
+  width: 30%;
+}
+
+.right-b-section {
+  width: 30%;
+}
+
+.right-b-section,
+.left-b-section {
+  padding: 8px 10px;
+}
+
+.footer-recommended-header {
+  margin-top: -30px;
+}
+
+.right-b-section ul {
+  list-style: none;
+}
+
+.right-b-section li {
+  margin: 5px 0;
+  padding: 3% 0;
+  font-size: 80%;
+  border-top: solid 1px rgb(82, 82, 82);
+}
+
+.bottom-section .left-b-section {
+  float: left;
+}
+
+.bottom-section .right-b-section {
+  float: right;
+}
```

Comment:
## No inefficient selectors[1]

- Try writing directly
```
.right-b-section {
  float: right;
}
```

###### _[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_